index.js

```diff
@@ -0,0 +1,56 @@
+
+function calcTotalSalary(lineup) {
+  const totalSalary = lineup.reduce((salary, player) => {
+    return salary + player.salary
+  }, 0)
+
+  return totalSalary <= 45000
+}
+
+function getTeamCount(lineup) {
```

Comment:
This name is a bit misleading. The function is really checking if you have too many players from a single team, not getting the count of teams. `violatesTeamSizeLimit` or something would be better.